examples/enable/basic_draw.py

```diff
@@ -19,16 +19,20 @@ def _draw_mainlayer(self, gc, view_bounds=None, mode="default"):
             gc.fill_path()
 
 
-class MyFrame(DemoFrame):
+class Demo(DemoFrame):
 
-    def _create_window(self):
+    def _create_component(self):
         box = Box(bounds=[100.0, 100.0], position=[50.0, 50.0])
         container = Container(bounds=[500, 500])
         container.add(box)
-        return Window(self, -1, component=container)
+        return container
+
+    def _create_window(self):
```

Comment:
This looks almost identical in every example: can it be moved to `DemoFrame`? In fact it may only be needed for `pyglet` version of `DemoFrame`.

Reply:
Yes, will do that.